components/rating/rating.scss

```diff
@@ -0,0 +1,12 @@
+.ui-rating {
```

Comment:
It could yes... But for consistency purposes and until we change to .css on all, I'd rather let this as it is.